examples/plot_season_summary.py

```diff
@@ -0,0 +1,168 @@
+"""
+Season Summary Visualization
+==================================
+This example demonstrates how to make an interactive season summarization
+dashboard showing points scored by each driver at each round.
+"""
+
+import pandas as pd
+import plotly.graph_objects as go
+from plotly.io import show
+from plotly.subplots import make_subplots
+
+import fastf1 as ff1
+
+
+##############################################################################
+# Fetch the event schedule. Skip the first row, which is the pre-season test.
+season = 2024
+schedule = ff1.get_event_schedule(season).iloc[1:]
```

Comment:
``get_event_schedule`` has an optional ``inlcude_testing`` argument that can be set to False and should be used instead